.github/actions/build-helm/action.yaml

```diff
@@ -1,8 +1,8 @@
 name: Build Helm packages
 description: Builds the helm packages
 inputs:
-  version:
-    description: The version of the operator that should be deployed
+  version_without_prefix:
```

Comment:
Would rather stay consistent here, as the calling workflow has 2 different variables too. 

If we don't to that it might lead future developers into using the wrong version.